src/lando/main/models/landing_job.py

```diff
@@ -281,31 +150,15 @@ def job_queue_query(
             grace_seconds (int): Ignore landing jobs that were submitted after this
                 many seconds ago.
         """
-        q = cls.objects.filter(status__in=JobStatus.pending())
-
-        if repositories:
-            q = q.filter(target_repo__in=repositories)
+        q = super().job_queue_query(repositories)
```

Comment:
This reorders how we build the QuerySet a bit. I'm assuming that the build order doesn't impact the final query when dealing with different parts (e.g., WHERE vs. ORDER BY).

Reply:
I think assumption is correct.